Tasks/AutomatedAnalysisV0/task.json

```diff
@@ -0,0 +1,468 @@
+                "range": "range"
+            },
+            "groupName": "orchestrationInputs",
+            "helpMarkDown": "If realtime, analysis is performed over a time duration beginning at the moment of execution. If range, analysis is performed over an explicitly specified time interval in the past"
+        },
+        {
+            "name": "windowType",
+            "type": "pickList",
+            "label": "Window type",
+            "defaultValue": "sliding",
+            "required": true,
+            "options": {
+                "sliding": "sliding",
+                "growing": "growing"
+            },
+            "groupName": "orchestrationInputs",
+            "helpMarkDown": "If growing, metrics during the entire lifetime is considered for evaluation. If sliding, metrics during the most recent interval used for evaluation."
+        },
+        {
+            "name": "delay",
+            "type": "string",
+            "label": "Delay",
+            "defaultValue":"0s",
+            "required": false,
+            "groupName": "orchestrationInputs",
+            "visibleRule": "analysisType == realtime",
+            "helpMarkDown": "Time (provide unit of time (s, m, h)) to wait before commencing collection and analysis of metrics"
+        },
+        {
+            "name": "lifetime",
+            "type": "string",
+            "label": "Lifetime",
+            "defaultValue": "10m",
+            "required": false,
+            "groupName": "orchestrationInputs",
+            "visibleRule": "analysisType == realtime",
+            "helpMarkDown": "Time (provide unit of time (s, m, h))) spanning the entire duration of the canary analysis"
+        },
+        {
+            "name": "analysisBeginTime",
+            "type": "string",
+            "label": "Analysis begin time",
+            "required": false,
+            "groupName": "orchestrationInputs",
+            "visibleRule": "analysisType == range",
+            "helpMarkDown": "Start time to collect the metrics. Provide timestamp in yyyy-mm-ddThh:mm:ssZ format (UTC time)"
+        },
+        {
+            "name": "analysisEndTime",
+            "type": "string",
+            "label": "Analysis end time",
+            "required": false,
+            "groupName": "orchestrationInputs",
+            "visibleRule": "analysisType == range",
+            "helpMarkDown": "Time to stop the metric collection. Provide timestamp in yyyy-mm-ddThh:mm:ssZ format (UTC time)"
+        },
+        {
+            "name": "interval",
+            "type": "string",
+            "label": "Interval",
+            "defaultValue": "2m",
+            "required": false,
+            "groupName": "orchestrationInputs",
+            "helpMarkDown": "single canary analysis task’s lifetime is composed of multiple loops of metrics collection and evaluation – each happening over a time window as specified under the interval input  (provide unit of time (s, m, h))"
```

Comment:
`Single canary analysis task’s lifetime is composed of multiple loops of metrics collection and evaluation – each happening over a time window as specified under the interval input (provide unit of time (s, m, h))`